templates/templates.go

```diff
@@ -3,9 +3,10 @@ package templates
 import (
 	"bytes"
 	"errors"
-	"io/ioutil"
+
```

Comment:
can we remove it everywhere in this project :pray: 
```
templates/templates_test.go:4:2: SA1019: package io/ioutil is deprecated: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details. (staticcheck)
```

Reply:
Removed

Reply:
:bow: